pkg/webhook/validation.go

```diff
@@ -43,14 +42,7 @@ func valid(ctx context.Context, ref name.Reference, rekorClient *client.Rekor, k
 		if err != nil {
 			return nil, err
 		}
-		sps, err := validSignaturesWithFulcio(ctx, ref, fulcioRoots, nil /* rekor */, nil /* no identities */, opts...)
-		if err != nil {
-			return nil, err
-		}
-		if len(sps) > 0 {
-			return sps, nil
-		}
-		return nil, errors.New("no valid signatures were found")
+		return validSignaturesWithFulcio(ctx, ref, fulcioRoots, nil /* rekor */, nil /* no identities */, opts...)
```

Comment:
It turns out that when `sps` is empty, this always returns an error, so the flow we had was pointless.

Reply:
superduper.

Reply:
👍🏻